backends/platform/sdl/sdl-window.cpp

```diff
@@ -168,10 +168,11 @@ void SdlWindow::grabMouse(bool grab) {
 }
 
 void SdlWindow::setMouseRect(const Common::Rect &rect) {
```

Comment:
I'm not exactly sure in which coordinates (window/drawable area) the parameter is expected to be in. Thus, I followed what was done in `SdlWindow::warpMouseInWindow()`, which expects drawable area coordinates, although its documentation refers to window coordinates. Maybe the terminology is not fully harmonized after the HiDPI mode was introduced.